docs/docs/09-typescript/02-inflights.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Inflight Functions
+id: inflights
+---
+
+The concepts of `preflight` and `inflight` in winglang are described [here](../02-concepts/01-preflight-and-inflight.md).
+In TypeScript, this is analogous to the difference between the `main` function (`preflight`) and the `inflight` function.
+
+### Functions and `inflight`
+
+Some resources, like `cloud.Function`, run code in the cloud. This code is an `inflight`.
+
+To create an `inflight` in TypeScript, you can use the `inflight` function and pass it an async closure:
+
+```ts
+import { main, cloud, inflight } from "@wingcloud/framework";
+
+main((root) => {
+  new cloud.Function(
+    root,
+    "MyFn",
+    inflight(async () => {
+      // This code runs in the deployed cloud function
+      return "Wing!";
+    })
+  );
+});
+```
+
+### `lift`ing `inflight`
+
+`inflight`s do not run in the same environment as the rest of the code, so you can't just reference variables from the outside.
```

Comment:
```suggestion
Inflight closures do not run in the same environment as the rest of the code, so you can't just reference variables from the outside.
```

(I would say there's no need to backtick every time we mention the term inflight, only when it refers to the actual code symbol)